src/test/skript/tests/syntaxes/expressions/ExprItemOwner.sk

```diff
@@ -0,0 +1,23 @@
+test "item owner":
+	###
+		Combines:
+			ExprItemOwner
+			ExprItemThrower
+			EffItemLifetime
+			CondItemLifetime
+	###
+	drop a stone at test-location
+	set {_drop} to last dropped item
+
+	set {_owner} to "Sahvde" parsed as offline player
+	set the item owner of {_drop} to {_owner}
+	assert the item owner of {_drop} is {_owner} with "Item Owner was not set to Sahvde"
+
+	set {_thrower} to "Fusezion" parsed as offline player
```

Comment:
In my opinion player names should be used here.
If either of those two people were to change their MC name, this test would fail?!?!?

Maybe just spawn some random entity and make them the owner/thrower.